cmd/account/rotate-secret.go

```diff
@@ -301,15 +310,14 @@ func (o *rotateSecretOptions) run() error {
 		}
 
 		if isSSSynced {
-			fmt.Printf("Sync completed...")
+			fmt.Println("Sync completed...")
 			break
 		}
 
-		fmt.Printf("Sync not completed, sleeping 5 seconds and rechecking...")
 		time.Sleep(time.Second * 5)
```

Comment:
Would probably be a good idea to at least keep some kind of output here, just so that it doesn't look like the command is hung.

Even just something as simple as `fmt.Print(".")` would probably suffice.

Reply:
I have opt'd for adding the printing of a . adding on to the previous print statement.